packages/micro.rb

```diff
@@ -3,26 +3,26 @@
 class Micro < Package
   description 'A modern and intuitive terminal-based text editor'
   homepage 'https://micro-editor.github.io/'
-  version '1.4.1'
+  version '2.0.13'
   license 'MIT'
   compatibility 'all'
   binary_compression 'tar.xz'
```

Comment:
also just curious about this line, it doesn't really matter that source is `tar.gz` in this case, right?

Reply:
That should really be this:
```
binary_compression tar.zst
```
But that's only to indicate what compression format the packaged binaries (from a build) have been uploaded with so that `crew` knows to figure out the download URL.

If you have `no_compile_needed` it shouldn't matter. (But do change it to the default of `tar.zst`...)